app/controllers/public/sessions.js

```diff
@@ -20,11 +19,7 @@ export default class SessionsController extends Controller {
 
   @action
   sessionChanged() {
-    if (this.model.isFiltering) {
-      $('html,body').animate({
-        scrollTop: $('#session-heading').offset()?.top
-      });
-    }
+    this.set('preserveScrollPosition', true);
```

Comment:
why created this.

Reply:
After only removing session changed whenever we click on filter page will reload and moves up using previous scoll position this well save the earlier scroll postion of page and didnt moves the page up and down

Reply:
you are right. :+1:

Reply:
good work @akash2408